src/astro/databases/base.py

```diff
@@ -308,6 +308,7 @@ def load_file_to_table(
             input_file.path,
             input_file.conn_id,
             normalize_config=normalize_config,
+            filetype=input_file.type.name,
```

Comment:
Should we add a unit-test for this case or not needed, wdyt?

Reply:
Added.